frontend/src/views/Game.tsx

```diff
@@ -1,31 +1,68 @@
 import Editor from '@monaco-editor/react';
 import React, { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Problem, getProblems } from '../api/Problem';
 import { Room } from '../api/Room';
+import { Text, ProblemHeaderText } from '../components/core/Text';
+import Header from '../components/navigation/Header';
 
 type LocationState = {
   room: Room,
 }
 
-function GamePage() {
+const FlexContainer = styled.div`
+  display: flex;
+  top: 50px;
+`;
+
+const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
+  overflow: none;
+  height: 100vh;
+  width: 50vw;
+`;
+
+export default function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
 
+  const [problems, setProblems] = useState<Problem[] | null>(null)
+
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((response) => {
+      setProblems(response);
+    }).catch((error) => {
+      console.log(error);
+    });
   }, [location]);
 
+  const firstProblem = problems?.[0];
```

Comment:
Add error handling if there are no problems?

Reply:
The error handling is supposed to be represented by line 42. Here the question mark accounts for the possibility of problems not existing. When I test this with only `npm start` so that it can't access the problem data, it prints an error to the console. Other than changing the error validation as in the above comment, are there any other ways you'd recommend doing this?

Reply:
I'd recommend displaying error text to show that an error occurred when trying to get something to load: see what @alankbi did in his PR [here](https://github.com/rocketden/main/pull/24/files#diff-c47bfa9c3f1b80608df1f7a1dc057e30R48), for example.

Reply:
Just put in a proper error handling system; I tried to make it as consistent as possible with Alan's in `Landing.tsx` though there are a few small differences.

Reply:
As discussed in the call, I think we want to have some error check in case zero problems are returned (an empty list). As for how to go about that, not too sure, but a proposed solution would maybe be in the `getProblems` then block, add another else-if to see if `problems[]` is empty (if so call setError with a custom message), and if it's not, then just save the first problem in state (rather than the whole list).

Reply:
I just made a change where if the problem set is an empty array in `Game.tsx` then it displays a different error message. It doesn't seem like good practice to put if/else statements in the then block in getProblems(), if that's even possible.

Reply:
Sounds good; also, I deleted all problems in the database to see how `problems?.[0]` would behave and everything seems to be good 👍